miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppSecureStorage.kt

```diff
@@ -1,22 +1,22 @@
 package com.rakuten.tech.mobile.miniapp.storage
 
 import android.content.Context
-import android.database.sqlite.SQLiteException
 import androidx.annotation.NonNull
 import androidx.annotation.VisibleForTesting
 import com.rakuten.tech.mobile.miniapp.errors.MiniAppSecureStorageError
 import com.rakuten.tech.mobile.miniapp.js.DB_NAME_PREFIX
-import com.rakuten.tech.mobile.miniapp.storage.database.DATABASE_BUSY_ERROR
-import com.rakuten.tech.mobile.miniapp.storage.database.DATABASE_UNAVAILABLE_ERROR
-import com.rakuten.tech.mobile.miniapp.storage.database.DATABASE_SPACE_LIMIT_REACHED_ERROR
 import com.rakuten.tech.mobile.miniapp.storage.database.MiniAppSecureDatabase
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
+import net.sqlcipher.database.SQLiteFullException
 import java.io.IOException
 import java.sql.SQLException
 
-@Suppress("TooManyFunctions", "LargeClass")
+@Suppress(
+    "LargeClass",
+    "TooManyFunctions"
+)
```

Comment:
Could you inline those suppress parameters?

Reply:
Addressed.